go.mod

```diff
@@ -7,16 +7,15 @@ require (
 	github.com/getsentry/sentry-go v0.13.0
 	github.com/go-git/go-git/v5 v5.4.2
 	github.com/gosuri/uilive v0.0.4
-	github.com/hexops/gotextdiff v1.0.3
```

Comment:
Isn't this required by the "smart deploy" feature? Did you update the latest master branch before doing this?